app/schemas/models/user.coffee

```diff
@@ -346,6 +346,16 @@ _.extend UserSchema.properties,
   role: {type: 'string', enum: ["God", "advisor", "parent", "principal", "student", "superintendent", "teacher", "technology coordinator"]}
   birthday: c.stringDate({title: "Birthday"})
   lastAchievementChecked: c.stringDate({ name: 'Last Achievement Checked' })
+  signupQueryParams: { type: 'object', properties: {
```

Comment:
Actually, it makes more sense for these to be top-level. I'm reorganizing these so the data fields can be repurposed later.

```
  israelId: {type: 'string', description: 'ID string used just for il.codecombat.com'}
  school: { 
    type: 'object', 
    description: 'Generic property for storing school information. Currently
                  only used by Israel; if/when we use it for other purposes,
                  think about how to keep the data consistent.',
    properties: {
      name: { type: 'string' }
      city: { type: 'string' }
      district: { type: 'string' }
      state: { type: 'string' }
      country: { type: 'string' }
    }
  }
```

What do you think?